packages/grant-explorer/src/features/discovery/__tests__/LandingPage.test.tsx

```diff
@@ -8,22 +8,32 @@ import {
   renderWithContext,
 } from "../../../test-utils";
 import { RoundMetadata } from "../../api/round";
-import {
-  RoundOverview,
-  getActiveRounds,
-  getRoundsInApplicationPhase,
-} from "../../api/rounds";
+import { RoundOverview } from "../../api/rounds";
 import LandingPage from "../LandingPage";
-import { MockedFunction, vi } from "vitest";
+import { vi } from "vitest";
 
 // Mock the API calls
-vi.mock("../../api/rounds", () => {
+
+// Create empty mock functions - we will set these inside the tests.
+const { graphql_fetch, fetchFromIPFS } = vi.hoisted(() => ({
+  graphql_fetch: vi.fn(),
+  fetchFromIPFS: vi.fn(),
+}));
+
+vi.mock("common", async () => {
+  const actual = (await vi.importActual("common")) as object;
```

Comment:
This contains a cast.

To mock with vi while preserving type safety, https://vitest.dev/guide/mocking.html says:


```ts
import { mocked, original } from './some-path.js'

vi.mock('./some-path.js', async () => {
  const mod = await vi.importActual<typeof import('./some-path.js')>('./some-path.js')
  return {
    ...mod,
    mocked: vi.fn()
  }
})
```

Reply:
Fixed